instrumentation/README.md

```diff
@@ -1,181 +1,247 @@
+on to [Splunk APM](https://docs.splunk.com/Observability/apm/intro-to-apm.html).
+
+For Linux distributions that do ***not*** support Debian/RPM packages or Java applications ***not*** running as systemd
+services, see the [Instructions for app servers](
+https://docs.splunk.com/Observability/gdi/get-data-in/application/java/instrumentation/java-servers-instructions.html)
+for how to manually enable and configure the Java agent.
+
+> ***Note***: The `libsplunk.so` shared object file and related enablement/configuration processes are
+> ***deprecated***.  The `splunk-otel-auto-instrumentation` deb/rpm package is also ***deprecated*** and replaced by the
+> `splunk-otel-systemd-auto-instrumentation` deb/rpm package.
+> 
+> [Installation](#installation) of `splunk-otel-systemd-auto-instrumentation` will automatically:
+> - Uninstall the `splunk-otel-auto-instrumentation` deb/rpm package (if installed).
+> - Remove `/usr/lib/splunk-instrumentation/libsplunk.so` from `/etc/ld.so.preload` (if it exists).
+> - Enable the Java agent ***only*** for systemd services via the
+>   `JAVA_TOOL_OPTIONS-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar` systemd environment
+>   variable.
+>
+> For reference only, the documentation for `libsplunk.so` has been moved to [libsplunk.md](./libsplunk.md).
+
+## Prerequisites/Requirements
+
+- Check [Java agent compatibility and requirements](
+  https://docs.splunk.com/Observability/gdi/get-data-in/application/java/java-otel-requirements.html)
+- Debian or RPM based Linux distribution with the `systemd` service manager
+- Java application(s) running as systemd services
+- [Install](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html) the Splunk OpenTelemetry
+  Collector
+
+## Installation
+
+The `splunk-otel-systemd-auto-instrumentation` deb/rpm package provides the following files to enable and configure the
+Java agent for systemd services:
+- [`/etc/systemd/system.conf.d/00-splunk-otel-javaagent.conf`](#systemd-environment-variables): Drop-in file with the
+  following default environment variables for systemd services:
+  - `JAVA_TOOL_OPTIONS=-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar`
+  - `OTEL_JAVAAGENT_CONFIGURATION_FILE=/usr/lib/splunk-instrumentation/splunk-otel-javaagent.properties`
+- `/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar`: The
+  [Splunk OpenTelemetry Auto Instrumentation Java Agent](https://github.com/signalfx/splunk-otel-java).
+- [`/usr/lib/splunk-instrumentation/splunk-otel-javaagent.properties`](#configuration-file): The default system
+  properties file to [configure the Splunk OpenTelemetry Auto Instrumentation Java Agent](
+  https://docs.splunk.com/Observability/gdi/get-data-in/application/java/configuration/advanced-java-otel-configuration.html).
+
+These packages can be installed from [package repositories](#debian-and-rpm-repositories) or downloaded from
```

Comment:
Suggestion to use a more active sentence: "Install these packages from... or download them from..."